crates/runtime/tests/docker/mod.rs

```diff
@@ -145,7 +146,10 @@ pub struct ContainerRunner<'a> {
 }
 
 impl<'a> ContainerRunner<'a> {
-    pub async fn run(self) -> Result<RunningContainer<'a>, anyhow::Error> {
+    pub async fn run(
+        self,
+        timeout: Option<Duration>,
```

Comment:
Oracle container initialization time is ~50s, added  optional timeout param to extend initialization time for Oracle to 2m to ensure tests are robust